treeherder/webapp/wsgi.py

```diff
@@ -18,7 +18,9 @@
 
 """
 import os
+from django.conf import settings
```

Comment:
I think you need to have DJANGO_SETTINGS_MODULE setup in your environment before you can safely import settings